models/points.go

```diff
@@ -1421,17 +1425,14 @@ func parseNumber(val []byte) (interface{}, error) {
 }
 
 func newFieldsFromBinary(buf []byte) Fields {
-	fields := Fields{}
+	fields := make(Fields, 5)
```

Comment:
Just better than not allocating anything.  It showed up as an extra allocation in profiling.